app/controllers/devise_overrides/sessions_controller.rb

```diff
@@ -1,12 +1,22 @@
 class DeviseOverrides::SessionsController < Devise::SessionsController
   def create
-    if params[:user_return_to].present?
-      session[:user_return_to] = params[:user_return_to]
-    end
-    super
-    if params[:rsvp_for].present?
-      lesson = Lesson.find(params[:rsvp_for])
-      current_user.rsvp_for(lesson)
+    if request.format.json?
+      resource = warden.authenticate(scope: resource_name)
```

Comment:
Useless assignment to variable - `      resource`.<br>Use 2 (not -4) spaces for indentation.<br>Use snake_case for variable names.